FlowCrypt/Functionality/DataManager/SessionService.swift

```diff
@@ -112,22 +110,6 @@ extension SessionService: SessionServiceType {
         return try switchActiveSession(for: currentUser)
     }
 
-    func logOutUsersThatDontHaveAnyKeysSetUp() throws {
```

Comment:
Is this no longer wanted, or has it moved somewhere?

Reply:
It wasn't used anywhere, so I just removed unused code

Reply:
It was called just a few lines above. But the question is, did it have any useful effect? That's hard to say.

Reply:
It wasn't called there, it was just in the list of `protocol SessionServiceType` methods - https://github.com/FlowCrypt/flowcrypt-ios/blob/8d1e041872723730fe0c44b1805e2fc78fae57fa/FlowCrypt/Functionality/DataManager/SessionService.swift#L39